src/iquery.cpp

```diff
@@ -0,0 +1,178 @@
+#include <irods/genquery2.h>
+
+#include "irods/table_column_key_maps.hpp"
+
+#include <irods/client_connection.hpp>
+#include <irods/irods_at_scope_exit.hpp>
+#include <irods/irods_exception.hpp>
+#include <irods/procApiRequest.h>
+#include <irods/rcMisc.h>     // For set_ips_display_name()
+#include <irods/rodsClient.h> // For load_client_api_plugins()
+
+#include <boost/program_options.hpp>
+#include <fmt/format.h>
+
+#include <algorithm>
+#include <cstdlib>
+#include <iostream>
+
+auto print_usage_info() -> void;
+auto print_columns_info() -> void;
+
+int main(int _argc, char* _argv[]) // NOLINT(modernize-use-trailing-return-type)
+{
+    set_ips_display_name("iquery (experimental)");
```

Comment:
Should we remove `(experimental)`?

Reply:
i think so - it's been proven/tested in the field and is the future.

Reply:
Done